### DISCUSSION TITLE
Possibility to hear samples in GOodf

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/GoOdf#68] [Ideas]
As the possibility is given in Goodf to fill some tuning parameters, maybe I'd be possible to hear samples as we have now to save the .organ file, open in GO, tuning as we want and come back to Goodf to copy the right parameters to have the "good" .organ file ...

**Comment by larspalo** [comment]
The problem is that you need to have some kind of reference for all kinds of voicing. Thus it's better to do the voicing inside of GrandOrgue. I've just recently implemented the possibility to import in GOODF a previously exported settings file (.cmb) from GrandOrgue to extract the selected voicing information and thus merge it into the odf file. This is not available in a release yet as I need to test it further, but it's already available for those who build GOODF themselves.

**Reply by Jean-Gabriel1982** [reply]
Ah ok i understand ! 
Indeed it'd be nice to import cmb file from GO !

**Comment by Bumblebee001** [comment]
Did I read somewhere that importation of cmb data is now possible?

**Reply by larspalo** [reply]
@Bumblebee001 Yes it's possible to import .cmb data to an .organ file with GoOdf, but with some limitations (only the voicing data, and it's only working reliably for exactly the same .organ+.cmb generation combination). The screencast nr 9 https://youtu.be/oAx4VU6-C94?feature=shared should have the details.
